packages/deployment/upgrade-test/upgrade-test-scripts/agoric-upgrade-10/legacy.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
```

Comment:
Makes sense. I was just confused about how it gets used because there's no string match for `legacy.sh`. Turns out its https://github.com/Agoric/agoric-sdk/blob/bf648099404393134febcf50444f7d52d9988a5b/packages/deployment/upgrade-test/upgrade-test-scripts/start_to_to.sh#L25

I think that's a fine way to support the non-JS way while we flesh out the JS way. I expect we'll get it all to one way eventually but it's good we can do it on whatever timeline